replace shell drush command with drush_path

seth-shaw-unlv opened the pull request:
Drush was not found, using the existing drush_path variable instead. [Resolves CLAW issue 775](https://github.com/Islandora-CLAW/CLAW/issues/775).

seth-shaw-unlv committed:
replace shell drush command with drush_path

Drush was not found, using the existing drush_path variable instead.

seth-shaw-unlv commented:
Figures I put the issue in the wrong place...

seth-shaw-unlv committed:
add default drush_path

whikloj commented on defaults/main.yml:
```diff
@@ -1,5 +1,7 @@
 ---
 
+drush_path: /usr/local/bin/drush
```
@seth-shaw-unlv Can you add `openseadragon_` to the front of this variable please

seth-shaw-unlv replied:
Note that the geerlingguy.drupal and geerlingguy.drush roles set the drush_path for the environment, and we will want to use whatever they set it to be (which is why it works). 

So, if we change the local drush_path to openseadragon_drush_path in this role, then we should set openseadragon_drush_path to whatever the current drush_path value is, if it is set. Should that logic be done in tasks/main.yml or in defaults/main.yml ?

whikloj replied:
So we are setting `drush_path` which is from `geerlingguy.drupal` and/or `geerlingguy.drush`? Or are they setting it?

My guess is to be _ansible compliant_ we should define a variable `openseadragon_drush_path` and then in the claw-playbook group_vars define `openseadragon_drush_path: {{ drush_path }}`

I think that should work.

seth-shaw-unlv replied:
Currently, geerlingguy.drush is setting the variable and we are using what they set. So, taking your strategy (which is fine by me) we should state in the documentation that this variable needs to be set before it will work. I'll update that today.

seth-shaw-unlv replied:
Slight modification. Keeping the openseadragon_drush_path default as it exists, but adding a new task to main to override it if drush_path is currently set.

whikloj commented on tasks/config.yml:
```diff
@@ -1,15 +1,15 @@
 
 - name: Check if modules are enabled
-  shell: drush pm-info openseadragon --format=json
+  shell: "{{ drush_path }} pm-info openseadragon --format=json"
```
Then update all these to be `openseadragon_drush_path` too

Natkeeran commented:
Tested this, and seems to fix the issue.

seth-shaw-unlv committed:
replace drush_path with openseadragon_drush_path

Using openseadragon_drush_path but added a new task to override it with the value of drush_path if it is set.

Natkeeran commented:
Tested this.  Looks good.

whikloj commented:
@seth-shaw-unlv Can I or you close this PR as #4 replaced it.
